src/Serokell/Util/Text.hs

```diff
@@ -105,6 +106,9 @@ tripleF = later tripleBuilder
 listJson :: (Foldable t, Buildable a) => Format r (t a -> r)
 listJson = later listBuilderJSON
 
+listMapTuple :: (Traversable t, Buildable a, Buildable b) => Format r (t (a, b) -> r)
+listMapTuple = later mapBuilder
```

Comment:
Why `listMapTuple`, if `t (a, b)` already corresponds to just "map"?

Reply:
Are you proposing to call it `listMap`?

Reply:
Something like this seems more reasonable IMO

Reply:
Fixed